packages/flutter/lib/src/material/i18n/material_fr.arb

```diff
@@ -3,5 +3,15 @@
   "backButtonTooltip": "Arrière",
   "closeButtonTooltip": "Fermer",
   "nextMonthTooltip": "Mois Suivant",
```

Comment:
Should this be "Mois suivant" for consistent capitalisation with 'Mois précédent' below?